javascript/ql/test/query-tests/Quality/UnhandledStreamPipe/test.qlref

```diff
@@ -1,2 +1,2 @@
-query: Quality/UnhandledStreamPipe.ql
```

Comment:
Let's also rename the test folder.

And rename the `.qlref` file to `UnhandledErrorInStreamPipeline.qlref` as per convention.